src/components/Navbar/components/CurrentLocationWidget/CurrentLocationWidget.tsx

```diff
@@ -0,0 +1,68 @@
+import { weatherSuccessResponse } from 'mocks/mockData';
+import ContentLoader from 'react-content-loader';
+import { Weather } from 'models/Weather';
+import { Container, WeatherIcon, Description, DegreeSign } from './CurrentLocationWidget.styled';
+
+const CurrentLocationWidget = () => {
+  let isLoading = false;
+  const data: Weather | any = weatherSuccessResponse; // temporary solution in order not to use
```

Comment:
**question** why any? in this situation it always be Waether

Reply:
I forgot to change it after improving my code. It will be `const data: any = {}` because @bednarskimichal told me that using dummy data is incorrect